tgui/packages/tgui/interfaces/ConstructionForklift/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { useBackend } from '../../backend';
+import { Button, Image, Section } from '../../components';
+import { Window } from '../../layouts';
+
+export const ConstructionForklift = (props: any) => {
+  const { act, data } = useBackend<ForkliftData>();
+  const { materials, modules, cooldowns, hologram_count, active_module_data } =
+    data;
+
+  let build_select: JSX.Element | undefined;
+  if (active_module_data !== undefined) {
+    build_select = (
+      <BuildTargetSelect
+        selected={active_module_data.current_selected_typepath}
+        available_builds={active_module_data.available_builds}
+      />
+    );
+  }
+
+  const set_active_module = (module: string) => {
+    act('set-active', { new_module_ref: module });
+  };
+  let module_select = (
+    <ModuleSelect modules={modules} set_active_module={set_active_module} />
+  );
+
+  return (
+    <Window title="Forklift Management Console">
+      <Window.Content>
+        {module_select}
+        {build_select}
+      </Window.Content>
+    </Window>
+  );
+};
+
+const ModuleAction = (action: string, params: object) => {
+  const { act } = useBackend();
+  const act_params = {
+    ...params,
+    action: action,
+  };
+  act('module-action', act_params);
+};
```

Comment:
not a fan of this, just call act directly in my opinion.

also this is not a component, no need to pascal case. components return JSX

`params: object` 
you're looking for `Record<string, any>`, objects, sadly, are not of type object